models/tmp/stg_shopify__abandoned_checkout_discount_code_tmp.sql

```diff
@@ -1,7 +1,7 @@
 {{ config(enabled=var('shopify_using_abandoned_checkout', True)) }}
 
 {{
-    fivetran_utils.union_data(
+    shopify_union_data(
```

Comment:
For consistency, can we include the namespace dispatch here as well?
```suggestion
    shopify_source.shopify_union_data(
```

Reply:
Then apply this for all the other calls.

Reply:
Applied.